thirdeye/thirdeye-frontend/app/pods/components/alert-details/component.js

```diff
@@ -496,7 +496,9 @@ export default Component.extend({
       isPreviewMode,
       alertId
     } = this.getProperties('analysisRange', 'notifications', 'isPreviewMode', 'alertId');
-
+    const toastOptions = {
+      timeOut: 10000
```

Comment:
Would it be possible to define this in a single place?

Reply:
definitely.  I'll add it to an appropriate utils file

Reply:
Done